### PR TITLE
GiotoHun Pet Feeder

### PR DESCRIPTION
**Pull request by lnx85**
Add Support for GiotoHun Pet Feeder PE-A03

### CODE REVIEW

File: custom_components/tuya_local/devices/giotohun_pet_feeder.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  name: Feed
+  icon: "mdi:bowl"
+  dps:
+    - id: 3
+      name: value
+      type: integer
+      range:
+        min: 1
+        max: 12
+      optional: false
```
Comment (make-all) [inline custom_components/tuya_local/devices/giotohun_pet_feeder.yaml:16]: Leave out the "optional" when it is false.

**Review by make-all — COMMENTED** [review]
Is this significantly different than the Catit Pixi feeder, which has basically the same dps with an additional "Quick feed" button?

**Comment by lnx85** [comment]
> Is this significantly different than the Catit Pixi feeder, which has basically the same dps with an additional "Quick feed" button?

In my pet feeder there is a DPS 4 for feeder status (feeding / standby); in HA that DPS is converted into a binary_sensor that become ON when the feeder is releasing the food. It is useful to check if the food has been succesfull released.

**Comment by lnx85** [comment]
> > Is this significantly different than the Catit Pixi feeder, which has basically the same dps with an additional "Quick feed" button?
> 
> In my pet feeder there is a DPS 4 for feeder status (feeding / standby); in HA that DPS is converted into a binary_sensor that becomes ON when the feeder is releasing the food. It is useful to check if the food has been succesfull released.

You can probably add the same DPS in the Catit Pixi feeder with optional: true instead of using another config file.

**Comment by make-all** [comment]
Merged with Catit Pixi Smart feeder, as mentioned in the comment above - added optional Feeding sensor.

**Pull request closed without merging**